tfhe/src/high_level_api/integers/parameters.rs

```diff
@@ -13,7 +11,7 @@ pub trait EvaluationIntegerKey<ClientKey> {
 }
 
 /// Trait to mark parameters type for integers
-pub trait IntegerParameter: ParameterType {
+pub trait IntegerId: Copy + Default {
```

Comment:
why do you prefer the Id wording to the Parameter one in that case ? I know it's something I never quite understood from the HL API design given I never really edited code in there

Reply:
Originally, each type (FheUint8, 16, 32, etc) was meant to have its own parameters, so there was `FheUint{8,16,32}Parameters`. Each parameter had an associated `Id` `FheUint{8,16,32}d` that is a zero sized typed stored in the GenericInteger, and this Id has some behabiour attached to it (like how to retrieve the correct client/server key)

However things changed now all types share the same key, the same base block and same parameters, so it makes more sense to just keep the `Id` thing.

Reply:
Ok thanks, I think there are some docstrings needing update just above that struct in that case, not sure about other locations